modules/aca/office_booking.rb

```diff
@@ -124,6 +47,7 @@ def on_update
         self[:timeout] = setting(:timeout)
```

Comment:
What format is this in? If the front end expects milliseconds is it worth allowing specification in `5m30s`-esque format here then converting `UV::Scheduler.parse_duration`?

Reply:
`booking_cancel_timeout` will be used instead of `timeout` (removed). Currently the value is in seconds only. I agree that allowing the user to enter values like '1m2s' instead of '62' is more intuitive so I'll change the required format here and change the default setting value to make it clear that the user needs to enter durations in that format.